main.tex

```diff
@@ -1178,57 +1281,57 @@ \section{The future of the Astropy project}
-Removing \python 2 support will allow the use of new, \python 3-only features,
-will simplify the code base, and reduce the testing overhead for the package.
+removing the support for \python 2 \citep{ape10}: future versions of \astropypkg
+will only support \python 3.5 or higher.
+Removing \python 2 support will allow the use of new \python 3-only features,
+simplify the code base, and reduce the testing overhead for the package.
 \astropypkg version 3.0 is currently scheduled for January 2018.
 
 In the next major release after version 3.0, scheduled for
-mid-2018, the focus will be on optimization of the code and improved
-documentation.
-To prepare for this release, we are preparing software for testing, evaluating,
-and monitoring the performance of the code.
-Less functionality may be introduced in this release while the focus will be
-primarily on improved performance.
+mid-2018, the focus will be on algorithm optimization and
+documentation improvement.
+To prepare for this release, we are subjecting the core package to testing,
+evaluation, and performance monitoring.
+As a result, less new functionality may be introduced as a trade-off for
+better performance.
 
 Beyond the core code, the \astropy project is also further developing the
 \astropy-managed affiliated packages.
 While these may not be integrated into the \astropypkg core package, these
-projects provide code that is useful to the astronomical community and meet the
+projects provide code that are useful to the astronomical community and meet the
 testing and documentation standards of \astropy.
-Some of these new efforts includes an initiative to develop tools for
+Some of these new efforts include an initiative to develop tools for
 spectroscopy \citep[\package{specutils}, \package{specreduc}, \package{specviz}]
-{ape13}, integration of LSST software, and packages to support HEALPIX
+{ape13}, integration of LSST software, and support for HEALPIX
 projection. %\citep{astropy-healpix}
 %\inlinecomment{BMS}{TODO: uncomment references of packages once they are available}
 
 \subsection{Learn Astropy}
 % Adrian
 
-The \astropypkg core package documentation contains narrative descriptions of
-the package functionality along with detailed usage notes for functions,
+The \astropypkg core package's documentation contains narrative descriptions of
+the package's functionality, along with detailed usage notes for functions,
 classes, and modules.
-While useful as a reference and for more advanced \python users, it is not the
-right entry-point for all users or learning environments.
+While useful as a reference for more experienced \python users, it is not the
+proper entry-point for other users or learning environments.
 In the near future, we will launch a new resource for learning to use both the
 \astropypkg core package and the many packages in the broader \astropy
-ecosystem, under the name `\emph{Learn Astropy}.'
+ecosystem, under the name \emph{Learn Astropy}.
```

Comment:
Note: Not sure what the quotes are for. Other new terms in this paper don't have quotes, so I removed them.